ymodem/ymodem.go

```diff
@@ -133,6 +141,8 @@ func ModemSend(c io.ReadWriter, bs int, files []File) error {
 
 	pBars := mpb.New(mpb.WithWidth(64))
 	defer pBars.Wait()
+	//to unblock if files were not sent
+	defer abortBars()
```

Comment:
better to combine abort and wait into finalize func

```go
finalizeBars := func() {
	for i := range files {
		if !f[i].bytesBar.Completed() {
			f[i].bytesBar.Abort(true)
		}
	}
	pBars.Wait()
}
```

Reply:
fixed with closure